.github/workflows/build.yml

```
@@ -20,10 +21,24 @@ jobs:
           if [[ ${#err} != 0 ]]; then
             echo "$err" | while IFS= read -r file; do
               echo "::error file={"$file"},title={checking}::Formatted $file"
-            done
-            exit 1
+              done
+              exit 1
           fi
     - name: Build targets
       shell: nix develop .#ci -c bash -e {0}
       run: |
-          make
\ No newline at end of file
+          make mlkem
+          ./checksum.sh ./test/test_kyber512 880e1fe907682fb5a9029fc7da114131a0740f4f3328d6ddfae46728b5ac8db0
+          ./checksum.sh ./test/test_kyber768 2b32e7f5aa91fe47b0ca9a1d5aa70d7dc7c4ff1ffcb7e8967e51bf15cc5d2095
+          ./checksum.sh ./test/test_kyber1024 87da552c0878350fa7643a6b49996893d4dba4d8be2badb32c4298c5c2b512c6
```

Do I understand correctly that this is to make sure that the tests don't print `ERROR`?
I think it is more robust to test for the return code. 

```suggestion
          ./test/test_kyber512
          ./test/test_kyber768
          ./test/test_kyber1024
```

---

Actually, this is to make sure that when the internal checks are successful, then it will print out information. 

When error happen, return code is non 0, also the hash won't match. 

Return 0 -> print infor -> check hash -> success
Return non 0 -> check hash -> fail.

---

I think that is confusing. Better to just check for the return code.

---

done in  https://github.com/pq-code-package/mlkem-c-aarch64/pull/29/commits/b76247fcc61c6b0d1322cacfe120603d104524ff